SU2_CFD/src/solvers/CFEASolver.cpp

```diff
@@ -2788,6 +2784,12 @@ void CFEASolver::Solve_System(CGeometry *geometry, CConfig *config) {
   /*--- Solve or smooth the linear system. ---*/
 
   auto iter = System.Solve(Jacobian, LinSysRes, LinSysSol, geometry, config);
+  if (rank == MASTER_NODE){
+    if (iter >= config->GetDeform_Linear_Solver_Iter()){
+      cout<<"Reached maximum number of iterations in structural deformation solver"<<endl;
+    }
+  }
+
```

Comment:
Ah I forgot the other day, remove this please, there are outputs for linear solver residuals and iterations.